ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -117,6 +122,38 @@ Verify RHODS Installation
     ...                   label_selector=app.kubernetes.io/name=data-science-pipelines-operator
     ...                   timeout=400
   END
+  ${kserve} =    Is Component Enabled    kserve    ${DSC_NAME}
+  IF    "${kserve}" == "true"
+        Install ISV Operator From OperatorHub Via CLI    operator_name=${SERVICEMESH_OP_NAME}
+        ...    subscription_name=${SERVICEMESH_SUB_NAME}
+        ...    catalog_source_name=redhat-operators
+        Wait Until Operator Subscription Last Condition Is
+        ...    type=CatalogSourcesUnhealthy    status=False
+        ...    reason=AllCatalogSourcesHealthy    subcription_name=${SERVICEMESH_SUB_NAME}
```

Comment:
probably we could `Wait For Pods To Be Ready    label_selector=name=istio-operator
    ...    namespace=${DEFAULT_OP_NS}`